services/ui/src/components/EnvironmentTeaser/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Link from 'next/link';
+
+export default ({ environment, project }) => {
+  const productionLabel = environment.environmentType == 'production' ? <div>Prod</div> : '';
+  const environmentURL = `/project/${project}/${environment}`.replace(/\s+/g, '-').toLowerCase();
+  return (
+    <div key={environment.id}>
```

Comment:
`key` is only necessary when the component is inside a loop or you want to force a re-render. I think this situation is neither and can be removed.